### DISCUSSION TITLE
Welcome to aipan-netdisk-search Discussions!

### DISCUSSION THREAD

**Discussion by unilei** [unilei/aipan-netdisk-search#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HomeBao** [comment]
可以布置在虚拟主机上吗

**Reply by unilei** [reply]
可以，我觉得
